### PR TITLE
PLAT-409: fail if intolerable call with validated state and object have no such state

### PR DESCRIPTION
**Pull request by MikhailZhukov**
Intolerable call with validated state panic with 'Not implemented' if object have no such state

### CODE REVIEW

File: ledger-core/virtual/execute/execute.go [hunk 1]
```diff
@@ -273,6 +273,19 @@ func (s *SMExecute) stepWaitPendingConstructorFinished(ctx smachine.ExecutionCon
 		ctx.Log().Test(markerPendingConstructorWait{})
 		return ctx.Sleep().ThenRepeat()
 	}
+	// set descriptor then pending constructor is finished
```
Comment (ElSacramento) [inline ledger-core/virtual/execute/execute.go:276]: when finished?

File: ledger-core/virtual/integration/method_test.go [hunk 2]
```diff
@@ -1779,7 +1814,7 @@ func TestVirtual_Method_IntolerableCallChangeState(t *testing.T) {
 		p1        = server.GetPulse().PulseNumber
 		isolation = contract.MethodIsolation{
 			Interference: contract.CallIntolerable,
-			State:        contract.CallValidated,
+			State:        contract.CallDirty,
```
Comment (ElSacramento) [inline ledger-core/virtual/integration/method_test.go:1817]: Why don't you add a ValidatedState to the object? According to documentation, we have intolerable+validated, tolerable+dirty calls in R0.

File: ledger-core/virtual/execute/execute.go (hunk 1 above)
Comment (MikhailZhukov) [reply ledger-core/virtual/execute/execute.go]: fix

File: ledger-core/virtual/integration/method_test.go (hunk 2 above)
Comment (MikhailZhukov) [reply ledger-core/virtual/integration/method_test.go]: done

**Pull request merged**